lib/src/transaction/msg/staking/MsgCreateValidator.spec.ts

```diff
@@ -111,9 +111,9 @@ describe('Testing MsgCreateValidator', function () {
                 securityContact: 'hitesh.goel@crypto.com',
             },
             commission: {
-                rate: '0.1',
-                maxRate: '0.2',
-                maxChangeRate: '0.01',
+                rate: '10000000000',
+                maxRate: '20000000000',
+                maxChangeRate: '10000000000',
```

Comment:
Are the values be like the below instead? 
```suggestion
rate: '100000000000000000',
maxRate: '200000000000000000',
maxChangeRate: '10000000000000000',
```